modules/library/jdbc/src/test/java/org/geotools/jdbc/SessionCommandListenerTest.java

```diff
@@ -99,12 +140,10 @@ public void testExpandVariables() throws Exception {
     }
 
     @Test
-    public void testInvalid() throws Exception {
-        try {
-            new SessionCommandsListener("startSession('${user')", null);
-            fail("This should have failed, the syntax is not valid");
-        } catch (IllegalArgumentException e) {
-            // fine
-        }
+    public void testInvalidVariableDefinitionMissingClosingBracket() throws Exception {
+        Assert.assertThrows(
+                IllegalArgumentException.class,
+                () -> new SessionCommandsListener("select ${user from 1)", null));
+        conn.commands.clear();
```

Comment:
If the test pass, this line is actually unreachable. Maybe also put this clean- up in the ``@After`` block?